strings.json

```diff
@@ -287,7 +287,7 @@
     "de": "Einzahlung von Bargeld in das Portfolio aus Dividenden",
     "fr": "Dépôt d\\'espèces au portefeuille lors des dividendes",
     "tr": "Temettülerden portföye yatan nakit",
-    "zh-Hant-TW": "股息入金至投資組合"
+    "zh-Hant-TW": "股利入金至投資組合"
```

Comment:
Definition from https://www.sfb.gov.tw/en/home.jsp?id=69&parentpath=0,9&mcustomize= is "股息; 股利" so both might be okay

Reply:
Will go with your suggestion since I'm not a native speaker

Reply:
Yes, in Taiwan they are just same semantically.
I've changed it since I have added several translation on "股利" other place, so I change this for consistency.
However, there might be better to reuse "股息" and change other places to follow.
Please feel free to change them, thanks~

Reply:
OK got it, I'll change other places to 股利 to be consistent